doc/api/async_hooks.md

```diff
@@ -395,8 +397,8 @@ For example:
 
 ```js
 console.log(async_hooks.currentId());  // 1 - bootstrap
-fs.open(path, (err, fd) => {
-  console.log(async_hooks.currentId());  // 2 - open()
+fs.open(path, 'r', (err, fd) => {
+  console.log(async_hooks.currentId());  // 6 - open()
```

Comment:
Yeah, something is wrong with the `async_hooks.currentId()` this is supposed to be `2`.